primaryHandler.go

```diff
@@ -259,48 +259,58 @@ type handleWebhooksIn struct {
 
 type handleWebhooksOut struct {
 	fx.Out
+	AddWebhookHandler     http.Handler `name:"add_webhook_handler"`
```

Comment:
I can't comment high enough, but some fields can be removed from `handleWebhooksIn` - the API Router and Authenticate.